xmtp_db/migrations/2025-06-09-225910_create_local_commit_log/up.sql

```diff
@@ -0,0 +1,25 @@
+-- TODO: Note that commit hash could show up multiple times in the commit log.
+CREATE TABLE remote_commit_log (
+    "sequence_id" BIGINT NOT NULL PRIMARY KEY,
+    "group_id" BLOB NOT NULL,
+    "last_epoch_authenticator" BLOB,
+    -- Comes from the epoch_authentication secret
+    -- https://www.rfc-editor.org/rfc/rfc9420.html#section-8-13
+    "epoch_authenticator" BLOB NOT NULL,
+    -- 1 = Success, all other values are failures matching the protobuf
+    "result" INT NOT NULL,
+    -- Items below this line should be null unless result is success
+    "epoch_number" BIGINT
+);
+
+CREATE TABLE local_commit_log (
+    "sequence_id" BIGINT NOT NULL PRIMARY KEY,
+    "group_id" BLOB NOT NULL,
+    "epoch_authenticator" BLOB NOT NULL,
+    -- 1 = Success, all other values are failures matching the protobuf
+    "result" INT NOT NULL,
+    -- Items below this line are for debugging purposes
+    "epoch_number" BIGINT NOT NULL,
+    "sender_inbox_id" TEXT,
+    "sender_installation_id" BLOB
```

Comment:
@codabrink I just realized that this table might immediately help with @mchenani's investigation. Could we also include the commit type (update membership, etc) here? Then commit logs could be compared across forked installations to figure out which commit broke the group

Reply:
Could we also add an auto-incrementing sqlite ID separately to the sequence ID? And not make anything unique other than the sqlite ID? That might help @mchenani rule out any issues with commits being processed multiple times or out of order